### PR TITLE
[Sync] Native schema updates

### PR DESCRIPTION
**Pull request by alxarch**
### Background

- This is the first PR for automated _sync_ of `mage schemas:push`

### Changes

* Updates comments and descriptions on some fields

### Testing

* N/A

### CODE REVIEW

File: schemas/logs/aws/tests/cloud_trail_digest_tests.yml [hunk 1]
```diff
@@ -60,10 +62,9 @@ result: |
         ],
         "p_log_type": "AWS.CloudTrailDigest"
   	}
-results: []
```
Comment (kostaspap) [inline schemas/logs/aws/tests/cloud_trail_digest_tests.yml:63]: Why is this removed?

Comment (alxarch) [reply schemas/logs/aws/tests/cloud_trail_digest_tests.yml:63]: I set it to `omitempty`

Comment (alxarch) [reply schemas/logs/aws/tests/cloud_trail_digest_tests.yml:63]: We usually want `result`  not `results`.

**Review by kostaspap — APPROVED** [review]
